packages/client/test/sim/mainnet.spec.ts

```diff
@@ -48,26 +51,30 @@ describe('simple mainnet test run', async () => {
 
   const blockHashes: string[] = []
   // ------------Sanity checks--------------------------------
```

Comment:
rest of the changes in this file are mostly because of formatting triggered by adding sim run timelimit